client/db/types.go

```diff
@@ -358,6 +358,7 @@ type MatchMetaData struct {
 	// Stamp is the match time (ms UNIX), according to the server's 'match'
 	// request timestamp.
 	Stamp uint64
+	// TODO: ReceiveTime uint64 -- local time stamp for match age and time display
```

Comment:
I wanted to sleep on the match stamp issue because unlike order there's no times stamp from the client's machine, only the server stamp.

One possibility is when making the `core.Match` for the frontend, this `MatchMetaData.Stamp` can be compared to the order's `Stamp` to get a time-since-order offset and add that to the order's `SubmitTime` to get a `ReceiveTime` approximation for the match.

Obviously we could also start recording the ReceiveTime as well when the match is received, but I'm not sure that's worth storing in the DB.

Reply:
Fine either way, but I would put it in the `MatchAuth`.